hw20/ItemDeck.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <vector>
+
+class Item;
+
+class ItemDeck
+{
+public:
+	ItemDeck();
+	~ItemDeck();
+
+	std::vector<Item*> generateItems();
+
+private:
+	std::vector<Item*> m_itemsDataBase;
+    std::vector<Item*> m_heldItems;
```

Comment:
Please keep an eye on formatting. It's absolutely not critical but impacts general clarity of the code)

Reply:
One of those insignificant at first glance points that distinguishes professional and amateur.